scripts/gtk-bundle-from-msys2.sh

```diff
@@ -58,25 +58,33 @@ getpkg() {
 	fi
 }
 
-GLIB=$(getpkg glib2)
-ATK=$(getpkg atk)
-PANGO=$(getpkg pango)
-CAIRO=$(getpkg cairo)
-ADW=$(getpkg adwaita-icon-theme)
-GTK=$(getpkg $gtk)
-
-cat <<EOF
-Using:
-glib: $GLIB
-atk: $ATK
-pango: $PANGO
-cairo: $CAIRO
-adwaita $ADW
-$gtk: $GTK
+read -d' ' pkgs <<EOF
```

Comment:
instead of read, you could also simply use
```shell
pkgs="
libwinpthread
gcc-libs
libiconv
...
"
```
Works the same in the for loop.

But if you like the read better I don't mind.